app/renderer/js/shared/preventdrag.ts

```diff
@@ -5,13 +5,13 @@
 // It doesn't affect the compose box so that users can still
 // use drag and drop event to share files etc
 
-const preventDragAndDrop = () => {
+function preventDragAndDrop(): void {
```

Comment:
We should be consistent here i.e. with es6.

Reply:
This can be done as a follow up I think.

Reply:
Done!